.github/workflows/ci-cd.yml

```
@@ -1,318 +1,119 @@
+          kubectl get deployments -n ${{ env.KUBE_NAMESPACE }}
+          kubectl get pods -l app=simple-app -n ${{ env.KUBE_NAMESPACE }}
+
+      - name: Production health check
+        run: |
+          SERVICE_IP=$(kubectl get service simple-app-service -n ${{ env.KUBE_NAMESPACE }} -o jsonpath='{.spec.clusterIP}' || echo "N/A")
+          echo "Service IP: $SERVICE_IP"
+          echo "Production deployment completed successfully!"
+          echo "Access via: kubectl port-forward service/simple-app-service 3002:80 -n production"
+          echo "Starting port-forward for production environment..."
+          kubectl port-forward service/simple-app-service 3002:80 -n production --address=0.0.0.0 &
+          echo "Production available at: http://localhost:3002" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)